pootle/apps/pootle_language/models.py

```diff
@@ -33,7 +33,9 @@ class LiveLanguageManager(models.Manager):
     def get_queryset(self):
         return super(LiveLanguageManager, self).get_queryset().filter(
             translationproject__isnull=False,
-            project__isnull=True,
```

Comment:
> hiding unused languages and languages used only in disabled projects or in obsolete TPs.

Correct me if my understanding is wrong, but since admins can still access disabled projects/TPs, shouldn't we display for them languages used only in disabled projects too? Otherwise accessing such a TP would have a broken drop-down.